spells.yml

```diff
@@ -3577,6 +3577,28 @@ feign_death:
     - necromancy
     - ritual
     - wizard
+feral_frenzy:
+  casting_time: 1 standard action
+  components: V, S, M
+  concentration: true
+  description: |-
+    You summon a frenzied squirrel, rat, or other Tiny beast in the space of one Small or larger creature within range. The summoned beast uses the frenzied critter statblock and acts at the start of each of your turns.
```

Comment:
This is hard to judge without seeing the frenzied critter statblock.

Reply:
Oh yup. Someday it will be in the standard, but for now, it's:
```yml

frenzied_critter:
  name: Frenzied Critter
  size: Tiny
  type: Beast
  alignment: Unaligned
  ac: 12
  hp: 1
  speeds:
    walk: 20
    climb: 20
  ability_scores: [2, 14, 7, 2, 9, 5]
  traits:
    - name: Minion
      desc: The critter never takes damage from a missed attack or an effect that it successfully saved against, and nothing can change its hit point maximum or grant it temporary hit points.
  attacks:
    - name: Bite
      type: melee weapon
      properties: [unarmed]
      hit: 1 piercing damage

```